### PR TITLE
changed PR name and aligned chainsize endpoint syntax with the other …

### PR DESCRIPTION
**Pull request by jobarjo**
- changed PR name 
- aligned chainsize endpoint syntax with the other ones e.g. removing layers to only keep chainsize data:
`{
    chainsize: "256"
}`

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1,80 +1,46 @@
 FORMAT: 1A
-HOST: http://polls.apiblueprint.org/
+HOST: https://example.org
```
Comment (bneiluj) [inline apiary.apib]: why example ?

File: apiary.apib [hunk 2]
```diff
@@ -1,80 +1,46 @@
...
 
 # Cosmos API Stake Capital
 
-Polls is a simple API allowing consumers to view polls and vote in them.
+API allowing clients to view state and retrieve data from the network.
```
Comment (bneiluj) [inline apiary.apib]: cosmos state

File: apiary.apib [hunk 3]
```diff
@@ -1,80 +1,46 @@
...
-                    }
-                ]
-            }
\ No newline at end of file
+# GET /chain_size
+
+Get the size of the blockchain. 
+
+This info will be pushed from the node directly.
+
```
Comment (bneiluj) [inline apiary.apib:41]: response data type: Kbytes

File: apiary.apib (hunk 1 above)
Comment (jobarjo) [reply apiary.apib]: to be changed with the correct host address; what should it be?

File: apiary.apib (hunk 2 above)
Comment (jobarjo) [reply apiary.apib]: will put: "to view Cosmos' state and retrieve data from the network"

File: apiary.apib (hunk 3 above)
Comment (jobarjo) [reply apiary.apib:41]: ok

File: apiary.apib (hunk 1 above)
Comment (bneiluj) [reply apiary.apib]: `cosmos.stake.capital/api`

Comment (jobarjo) [reply apiary.apib]: ok

**Pull request merged**